### ISSUE TITLE
grades are not actioned the same as other module

### ISSUE THREAD

**Issue by barry-matthias** [h5p/moodle-mod_hvp#395]
Creating a graded h5p activity which generates a grade in the lms gradebook is not recovered if the teacher overrides the grade then removes the override tick box in the grade item.
![Screenshot_2021-03-16 Results for Unit 3 Assignment 9 Units of Angular Measurements](https://user-images.githubusercontent.com/21336311/111345526-46ad7300-8675-11eb-8755-412165cd2ae4.png)

Instead the grade is graded as null which does not reflect the attempt and grade report for the h5p module

/mod/hvp/grade.php shows there is an attempt for the activity however since the grade was overidden then the override was removed the original grade generated by the module is no longer present even tho the attempt report shows the original value.

**Comment by thomasmars** [comment]
Yes, when score is overridden it gets overwritten, do you think we should keep the original score in case the teacher wants to revert their override ?
